src/rag/gl/correcto.dic

```diff
@@ -39029,6 +39030,7 @@ grafómano/10,15 po:substantivo
 gramático/10,15 po:substantivo
 granxeiro/10,15 po:substantivo
 gravateiro/10,15 po:substantivo
+Groenlandia po:substantivo
```

Comment:
Hai que documentalo

Reply:
https://gl.wikipedia.org/wiki/Groenlandia
https://academia.gal/dicionario/-/termo/busca/groenland%C3%A9s

Reply:
OK, pero...
Non sei se te acordas daquel magnífico traballo que fixeron na TVG de toponimia universal, que traballaron realmente as formas galegas... esa é a documentación que eu buscaba.
Tanto a da RAG como aínda máis a Galipedia non son fontes especializadas no tema.
Aceptable en todo caso a documentación.